site/_posts/2021-09-15-rbvmomi-maintenance.md

```diff
@@ -0,0 +1,23 @@
+---
+title: RbVmomi Maintenance
+date: 2021-09-15 18:00 UTC
+author: agrare
+tags: collaboration announcements
+---
+
+I have the pleasure of announcing that the [ManageIQ community](https://www.manageiq.org/community/) will be taking over ongoing maintenance of the RbVmomi gem.
+
+RbVmomi is a Ruby SDK for the VMware vSphere API which allows users to manage their virtual infrastructure.
+
+Up until this point RbVmomi has been managed by VMware as a community supported project, but due to lack of Ruby maintainers it has since been archived [[ref]](https://github.com/vmware-archive/rbvmomi/commit/fb3caa94264827c27f00a7369090bd2421b82524).
+
+The ManageIQ community is picking up where VMware left off and will be providing security updates, bug fixes, and updates for new versions of VMware vSphere in a timely manner.
+
+The gem will continue to be available from rubygems at https://rubygems.org/gems/rbvmomi so if you are currently using RbVmomi you don't need to change anything.
```

Comment:
TBD pending gem owner access

Reply:
VMware has decided to not allow for the continued use of https://rubygems.org/gems/rbvmomi